src/Resources/translations/SonataPageBundle.nl.xliff

```diff
@@ -721,7 +721,7 @@
             </trans-unit>
             <trans-unit id="179" resname="shared_block">
                 <source>shared_block</source>
-                <target>shared_block</target>
+                <target>Blokken</target>
```

Comment:
I could make it into "Gedeelde blokken" if you want. However the nuance is slightly different, in the context (left menu) where this is used, it made more sense without the "shared" part and looks a little weird to me otherwise.
Actually if you translate it directly it isn't plural either, so it would be "Gedeelde blok", but that really wouldn't make sense to anyone.

Reply:
Thanks for the clarification, it was just an observation.